README.md

```diff
@@ -13,11 +13,14 @@ The current version is not considered stable.
 
 ## Table of Contents
 
-- [Install](#install)
-- [Usage](#usage)
-- [Contributing](#contributing)
-- [License](#license)
-- [Acknowledgements](#acknowledgements)
+- [ｎｅｐｔｕｎｅ](#ｎｅｐｔｕｎｅ)
```

Comment:
My markdown extension did this, I think it's nice.

Reply:
i does look cool but we don't have any other top level headers to it feels like we don't need 😬 am i a party pooper